runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -715,6 +746,55 @@ else if (activeTab != null
 			tagManager.removeTag(itemId, activeTab.getTag());
 			bankSearch.search(InputType.SEARCH, TAG_SEARCH + activeTab.getTag(), true);
 		}
+		else if (activeTab != null
+			&& event.getWidgetId() == WidgetInfo.BANK_ITEM_CONTAINER.getId()
+			&& event.getMenuAction() == MenuAction.RUNELITE
+			&& event.getMenuOption().startsWith(EDIT_QUANTITY))
+		{
+			// Add "edit-quantity" menu entry to all items in bank while tab is selected
+			event.consume();
+			final ItemComposition item = getItem(event.getActionParam());
+			final int itemId = item.getId();
+			final ItemComposition itemComposition = itemManager.getItemComposition(itemId);
+			final String name = itemComposition.getName();
+
+			final String initialValue = tagManager.getQuantityOfItemForTag(activeTab.getTag(), itemId, false);
+
+			chatboxPanelManager.openTextInput(name + " quantity:<br>(enter blank to reset)")
+				.addCharValidator(FILTERED_CHARS)
+				.value(initialValue)
+				.onDone((newValue) ->
+					clientThread.invoke(() ->
+					{
+
+						// Set quantity
+						try
+						{
+							//Convert stack supported input to quantity
+							long stackSizeToQuantity = QuantityFormatter.parseQuantity(newValue);
+							//Convert long to integer. Will throw instead of overflow
+							int newValueInteger = Math.toIntExact(stackSizeToQuantity);
```

Comment:
Why would you do this instead of keeping the to stack size in the string?

Reply:
Didn't think about doing it this way initially due to the original check to ensure the value was over 1. Since earlier I removed that, this way is possible and is definitely better, and will be changed in the next commit.